src/avalanchemq/vhost.cr

```diff
@@ -930,5 +930,23 @@ module AvalancheMQ
     def shovels
       @shovels.not_nil!
     end
+
+    def purge_all_queues!(max_count : Int? = nil) : UInt32
+      sum = 0_u32
+      @queues.each_value do |q|
+        sum += q.purge(max_count)
+      end
+      sum
+    end
+
+    def reset!(backup_data = true)
```

Comment:
Could we write some specs?

Reply:
specs are coming, just wanted to see your input first.

Reply:
... and I think this makes sense, just want to be sure what will actually happens to consumers with unacked messages etc. :)

Reply:
unacked will right now just be putted back in the queue, so need to find all consumers with unacked and clear them. Or maybe we can do something with just clear the `@unacked`  in `Queue` and when a consumers acks check if the tag is in @unacked.